packages/backend-modules/payments/graphql/resolvers/_queries/getOffer.ts

```diff
@@ -1,15 +1,19 @@
 import { GraphqlContext } from '@orbiting/backend-modules-types'
-import { Offers, Shop, utils } from '../../../lib/shop'
+import {
+  checkIntroductoryOfferEligieblity,
+  Offers,
+  Shop,
+} from '../../../lib/shop'
 
 export = async function getOffer(
   _root: never,
   args: { offerId: string; promoCode?: string },
   ctx: GraphqlContext,
 ) {
-  const withEntryOffer = ctx.user
-    ? (await utils.hasHadMembership(ctx.user.id, ctx.pgdb)) === false
-    : true // if there is no user we show the entry offers
-
+  const withEntryOffer = await checkIntroductoryOfferEligieblity(
```

Comment:
`withIntroductoryOffer` also here?

Reply:
I know all the places where it is inconsistent ;)
Will fix it.